web/src/main/java/org/cbioportal/web/parameter/StudyViewFilter.java

```diff
@@ -19,11 +19,12 @@
     private List<String> studyIds;
     private List<ClinicalDataEqualityFilter> clinicalDataEqualityFilters;
     private List<ClinicalDataIntervalFilter> clinicalDataIntervalFilters;
-    private List<MutationGeneFilter> mutatedGenes;
-    private List<CopyNumberGeneFilter> cnaGenes;
-    private Boolean withMutationData;
-    private Boolean withCNAData;
-    private RectangleBounds mutationCountVsCNASelection;
+	private List<MutationGeneFilter> mutatedGenes;
+    private List<FusionGeneFilter> fusionGenes;
```

Comment:
standardize indentation to uniform 4-space indents (no tabs)

Reply:
corrected indents.